src/QueryBuilder/ColumnConverter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\QueryBuilder;
+
+use Keboola\Datatype\Definition\Teradata;
+use Keboola\StorageDriver\Command\Table\ImportExportShared\DataType;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+
+class ColumnConverter
+{
+    public const DATA_TYPES_OPTIONS = [
+        DataType::INTEGER,
+        DataType::REAL,
+    ];
+
+    public const DATA_TYPES_MAP = [
+        DataType::STRING => Teradata::TYPE_VARCHAR,
+        DataType::INTEGER => Teradata::TYPE_INTEGER,
+        DataType::DOUBLE => Teradata::TYPE_DOUBLE_PRECISION,
+        DataType::BIGINT => Teradata::TYPE_BIGINT,
+        DataType::REAL => Teradata::TYPE_REAL,
+        DataType::DECIMAL => Teradata::TYPE_DECIMAL,
+    ];
+
+    /**
+     * Only cast STRING type to a given NUMERIC type
+     */
+    public function convertColumnByDataType(string $column, int $dataType): string
```

Comment:
chapu, je to asi nejlepsi reseni, ale i tak mi to prijde divny -> https://keboola.slack.com/archives/CFVRE56UA/p1669714494890299

Reply:
nechame tak, vyresime v https://keboola.atlassian.net/browse/KBC-3063